test/Microsoft.Extensions.Logging.Testing.Tests/AssemblyTestLogTests.cs

```diff
@@ -46,10 +46,13 @@ public void TestLogWritesToITestOutputHelper()
                 logger.LogTrace("Trace!");
             }
 
-            Assert.Equal(@"[TIMESTAMP] TestLifetime Information: Starting test TestLogWritesToITestOutputHelper
-[TIMESTAMP] TestLogger Information: Information!
-[TIMESTAMP] TestLifetime Information: Finished test TestLogWritesToITestOutputHelper in DURATION
-", MakeConsistent(output.Output), ignoreLineEndingDifferences: true);
+            var testLogContent = MakeConsistent(output.Output);
+
+            Assert.Equal(
+@"[XUNITTIMESTAMP] TestLifetime Information: Starting test TestLogWritesToITestOutputHelper at TIMESTAMP
```

Comment:
I did not update the xunit timestamp format that's displayed to the console upon failure. Do we want this too?

Reply:
Yes